### ISSUE TITLE
Unable to see events rendered in tests

### ISSUE THREAD

**Issue by tgandrews** [fullcalendar/fullcalendar-react#130]
### Reduced Test Case
1. Clone the repo https://github.com/tgandrews/fullcalendar-test
2. `yarn`
3. `yarn start` see the events rendered
4. `yarn test` see the tests pass but the snapshot include no events in `src/__snapshots__/App.test.js.snap`

### Bug Description

Describe how to recreate the bug.
Follow the steps above to be able to recreate. This works with v4 providing that is used instead.

What do you expect to happen?
I expect events to be rendered in test environments

What happens instead?
No events are rendered in tests even after waiting for a long time 20seconds in the linked example

**Comment by tgandrews** [comment]
Yes it works in the browser fine. Just not in the jsdom environment

**Comment by chunyinli** [comment]
Having the same issue while using resourceTimeGridPlugin. Tried stripping it down to a calendar with a single event passed via `events`, with no other options. Event shows up in browser, but cannot be found in test. If I change it to use the daygridPlugin, then it works in browser and in test environment.

**Comment by arshaw** [comment]
I will take a look at this very soon. This likely started happening because of my [event-positioning refactor](https://fullcalendar.io/blog/2021/05/new-features-in-v570). In a Jest environment there isn't a real DOM with real coordinates. In previous version of FullCalendar, events must have rendered even without available coordinates, whereas now it is likely waiting for non-zero coordinates as a precondition for rendering. I'll take a look.

**Comment by arshaw** [comment]
I believe I've found where rendering is short-circuiting when no available coordinates:

- timegrid: https://github.com/fullcalendar/fullcalendar/blob/v5.7.2/packages/timegrid/src/TimeCol.tsx#L122
- timeline: https://github.com/fullcalendar/fullcalendar-scheduler/blob/064431ef1212ad1cfb41776f0124302557e06325/timeline/src/event-placement.ts#L29
- daygrid (aka month view) seems unaffected

The next step is to modify the code so events are rendered unconditionally, even without coordinates, which may required a tiny bit of refactoring.

**Comment by arenac** [comment]
Thanks for the fast answer Adam.

Based on that new version my dependencies are like 

```
"@fullcalendar/react": "^5.7.0",
"@fullcalendar/resource-timeline": "^5.7.2",
```

But the issue is still there. I guess the next step is that refactoring you mentioned. Looking forward to it.

**Comment by arshaw** [comment]
this has been implemented and queued for release.
